test/sharness/t0125-twonode.sh

```diff
@@ -83,6 +83,12 @@ run_advanced_test() {
   test_expect_success "shut down nodes" '
     iptb stop && iptb_wait_stop
   '
+
+  # NOTE: data transferred stats checks are flaky
+  #       trying to debug them by printing out the stats hides the flakiness
+  #       my theory is that the extra time cat calls take to print out the stats
+  #       allow for proper cleanup to happen
+  go-sleep 1s
```

Comment:
Without the sleep the testcase was being flaky for me. I tried to investigate it further by adding some prints here and there but that always deflaked the test so it got me nowhere. In all honesty, I didn't spend too much time on this.